Gemfile.lock

```diff
@@ -101,11 +106,13 @@ PLATFORMS
 DEPENDENCIES
   html-proofer
   jekyll
+  json-schema
   mdl
   minima (~> 2.0)
+  toml
 
 RUBY VERSION
-   ruby 2.5.1p57
+   ruby 2.6.4p104
 
 BUNDLED WITH
-   1.16.6
+   1.17.3
```

Comment:
I'd suggest upgrading bundler as well:
```
~/projects/bitcoin/bitcoinops.github.io (pr/224)$ gem install bundler
Fetching bundler-2.0.2.gem
Successfully installed bundler-2.0.2
Parsing documentation for bundler-2.0.2
Installing ri documentation for bundler-2.0.2
Done installing documentation for bundler after 2 seconds
1 gem installed
```

Reply:
I'd prefer to leave that for later.  This PR resolves a problem we had, but subsequent to it getting merged, I plan to PR an update to the latest version of Jekyll that will update almost all of our dependencies.  To me, that seems like the correct time to update bundler.

Reply:
Bundler is the dependencies resolver/package manager, so I tend to update it before changing/upgrading dependencies. If helpful, I'm on the latest and had no issues with the deps when bundle installing, but as long as you're not seeing any issues then either way should be fine.